### PR TITLE
add colorization to e2e tests in Jenkins

### PR DESCRIPTION
**Pull request by kibbles-n-bytes**
Fixes the colorization in Jenkins' console output.

Closes: #844

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -121,13 +121,15 @@ node {
...
       """
 
-      // Run the e2e test framework
-      sh """${env.ROOT}/contrib/jenkins/run_e2e.sh \
-            --registry gcr.io/${test_project}/catalog/ \
-	    --version ${version} \
-	    --cleanup \
-	    --create-artifacts
-      """
+      ansiColor('xterm-darker-gray') {
```
Comment (pmorie) [inline Jenkinsfile]: Is this going to mean that the color codes still show up in the log?  I'd love a colorized jenkins log if I could get it, but I don't think I've ever seen a jenkins do that.

Comment (kibbles-n-bytes) [inline Jenkinsfile]: This actually fixes the colors; I installed a plugin that allowed for that support. So it should look pretty from the Console Output logs, and the Plain Text logs won't show any color codes.

It seems like it just detects the color codes, and then substitutes in a span in the HTML with appropriate styling.

Comment (pmorie) [inline Jenkinsfile]: That sounds pretty beast...

**Comment by pmorie** [comment]
Merging, because the output is correct and this makes it much easier to read

**Pull request merged**